docs/ref/templates/api.txt

```diff
@@ -554,17 +554,17 @@ settings file, the default template engine contains the following context
 processors::
 
     [
-        'django.contrib.auth.context_processors.auth',
         'django.template.context_processors.debug',
-        'django.template.context_processors.i18n',
-        'django.template.context_processors.media',
-        'django.template.context_processors.static',
-        'django.template.context_processors.tz',
+        'django.template.context_processors.request',
+        'django.contrib.auth.context_processors.auth',
         'django.contrib.messages.context_processors.messages',
     ]
 
 .. versionchanged:: 1.8
 
+    The request context processor was added to the default settings file and
```

Comment:
I'm not sure this note is really need as in Django < 1.8, the default settings generated by startproject didn't include any context processors.

Could you also change the sentence "Here's what each of the default processors does" on this page (default -> built-in)